lua/gen/init.lua

```diff
@@ -200,6 +200,13 @@ M.exec = function(options)
     if string.find(cmd, "%$body") then
         local body = {model = opts.model, prompt = prompt, stream = true}
         if M.context then body.context = M.context end
+        if M.model_options ~= nil then -- llamacpp servert - model options: eg. temperature, top_k, top_p
```

Comment:
```suggestion
        if M.model_options ~= nil then -- llamacpp server - model options: eg. temperature, top_k, top_p
```

Reply:
fixed in 93c5eadcdb19bfea21cf2767baa5a4345532f91f